src/marketplace-custom-config.js

```diff
@@ -64,3 +64,24 @@ export const dateRangeFilterConfig = {
 export const keywordFilterConfig = {
   active: true,
 };
+
+const relevanceKey = 'relevance';
```

Comment:
Add some comment here too. Since this a config file, it might even be better to keep the `relevanceKey: 'relevance'` inside the sortConfig variable.